pkg/app/piped/platformprovider/kubernetes/diff.go

```diff
@@ -60,13 +60,20 @@ func Diff(old, new Manifest, logger *zap.Logger, opts ...diff.Option) (*diff.Res
 	}
 
 	key := old.Key.String()
-	normalized, err := remarshal(new.u)
+
+	normalizedOld, err := remarshal(old.u)
+	if err != nil {
+		logger.Info("compare manifests directly since it was unable to remarshal Kubernetes manifest to normalize special fields", zap.Error(err))
```

Comment:
[nit] It would be nice to show old or new when investigating with this log.
It is the same for the remarshal new one.

```suggestion
		logger.Info("compare manifests directly since it was unable to remarshal old Kubernetes manifest to normalize special fields", zap.Error(err))
```

Reply:
Fixed by this commit
https://github.com/pipe-cd/pipecd/pull/5154/commits/ead1e14988b9982a4d541234d8d270e653bca194